_aggregations/pipeline/index.md

```diff
@@ -0,0 +1,209 @@
+
+Pipeline aggregations are run after all other sibling aggregations. This has performance implications. For example, using the `bucket_selector` pipeline aggregation to narrow a list of buckets does not reduce computations performed on omitted buckets.
+{: .note}
+
+Pipeline aggregations cannot be sub-aggregated, but can be chained with other pipeline aggregations. For example, you calculate a second derivative by chaining two consecutive `derivative` aggregations. Keep in mind that pipeline aggregations append to existing output. So for example computing a second derivative by chaining `derivative` aggregations outputs both the first and second derivatives.
+
+## Types of pipeline aggregations
+
+Pipeline aggregations are of two types, sibling and parent.
+
+### Sibling aggregations
+
+A sibling pipeline aggregation takes the output of a nested aggregation and produces new buckets or new aggregations at the same level as the nested buckets.
+
+A sibling aggregation must be a multi-bucket aggregation (have multiple grouped values for a certain field) and the metric must be a numeric value.
+
+`min_bucket`, `max_bucket`, `sum_bucket`, and `avg_bucket` are common sibling aggregations.
+
+### Parent aggregations
+
+A parent aggregation takes the output of an outer aggregation and produces new buckets or new aggregations at the same level as the existing buckets.
+
+The specified metric for a parent aggregation must be a numeric value.
+
+We strongly recommend setting `min_doc_count` to `0` (the default for `histogram` aggregations) for parent aggregations. If `min_doc_count` is greater than `0` the aggregation omits buckets, which might lead to incorrect results.
+{: .important}
+
+`derivatives` and `cumulative_sum` are common parent aggregations.
+
+## Buckets path property
+
+A pipeline aggregation uses the `buckets_path` property to access the results of other aggregations.
+The `buckets_path` property has the following syntax:
+
+```
+buckets_path = <AGG_NAME>[<AGG_SEPARATOR>,<AGG_NAME>]*[<METRIC_SEPARATOR>, <METRIC>];
+```
+
+where:
+
+- `AGG_NAME` is the name of the aggregation.
+- `AGG_SEPARATOR` separates aggregation names. It's represented as `>`.
+- `METRIC_SEPARATOR` separates the final aggregation from its metrics. It's represented as `.`.
+- `METRIC` is the name of the metric. It is required in the case of multi-value metric aggregations.
+
+For example, `my_sum.sum` selects the `sum` metric of an aggregation called `my_sum`. `popular_tags>my_sum.sum` nests `my_sum.sum` into the `popular_tags` aggregation.
+
+The following parameters are optional:
+
+- `gap_policy`: Real-world data can contain gaps or null values. You can specify the policy to deal with such missing data with the `gap_policy` property. You can either set the `gap_policy` property to `skip` to skip the missing data and continue from the next available value, or `insert_zeros` to replace the missing values with zero and continue running.
```

Comment:
I think we can link to the data gaps section here instead of repeating the info.

Reply:
Replaced this section with Data gaps and deleted the Data gaps section. What do you think?

Reply:
Looks good! We can then link to this section from other aggregation pages that use this info.